gen_binary.py

```diff
@@ -0,0 +1,60 @@
+from misc.viz_utils import visualize_instances_dict
+import numpy as np
+import json
+from skimage import io
+import cv2
+import zarr
+import sys, getopt, os
+
+'''
+Sample call: gen_binary.py -i "/mnt/ibm_sm/home/snigdha/TSP14 UB B2.json" -o /mnt/ibm_sm/home/snigdha/ -h 67343 -w 143424
```

Comment:
we will have 90 json files for the 90 svs files we have, so I suggest having a program that takes in a folder of all json files and if possible can infer the height, width from the json file or the svs file that comes along with it. 

Otherwise you would have to manually find out the heights and widths and run these commands for each json